### ISSUE TITLE
A11y_Oracle Assessment for Azure Data Studio (ADS)_AssessmentReport_SkuRecommendations_Keyboard:Tooltip present at "Sku Recommendations" is not accessible through keyboard. 

### ISSUE THREAD

**Issue by v-akr** [microsoft/azuredatastudio#19582]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHub Tags:
#A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win11;#DesktopApp;#WCAG2.1.1;#Benchmark;#FTP;#Keyboard;

### Environment Details:

Application Name: Oracle Assessment for Azure Data Studio (ADS)
Operating System: Windows 11
Version :1.35.1

### Repro Steps:

1. Download the azure data studio
2. Open the application and tab to the Connections icon in left navigation.
3. Tab to the new connections button and press enter. Connection dialog box will open.
4. Fill Correct details in connection details and tab to the connect button and press enter. New connection will be added.
5. Tab to the Database Migration Assessment for Oracle button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
6. Tab to the "Run New Assessment" button at top the page and press enter.New Database Migration Assessment for Oracle dialog box will open. Fill the correct details. Tab to "Start Assessment button and press enter.
7. New Assessment can be seen in last five assessment tab. Tab to the assessment press enter. New tab will get open.
8. Tab to Assessment overview by platforms section. Tab to Assessment for Azure virtual machine in the section.Tab to view report details button and press enter. New screen will open.
9. Tab to "Sku Recommendations" tab and press enter.
10. Navigate using keyboard and observe the tab items.

### Actual:

While navigating with keyboard, Tooltip present at the tab item "Sku Recommendations" is not accessible through keyboard.

Similar issue is observed for Feature Compatibility tab item.

### Expected:
Tooltip present at "Sku Recommendations" should be accessible through keyboard.

### User Impact:
Low vision user will not be able to get the complete information about the selected item.

**Comment by aasimkhan30** [comment]
This is an upstream issue belonging to https://github.com/microsoft/vscode/issues/141021.
As of now, they have decided to not go ahead and fix it.

**Comment by alanrenmsft** [comment]
@v-akr as per the process, please downgrade this issue to Sev3.

**Comment by alanrenmsft** [comment]
closing as the tester's alias is no longer availabe, will reactivate if the test team marks this issue properly.
